packages/indexer/src/vcr/client.ts

```diff
@@ -0,0 +1,73 @@
+import type {
+  Client,
+  ClientCallOptions,
+  StatusRequest,
+  StreamConfig,
+  StreamDataOptions,
+  StreamDataRequest,
+  StreamDataResponse,
+} from "@apibara/protocol";
+import type { VcrConfig } from "./config";
+import path from "node:path";
+import fs from "node:fs";
+import type { CassetteDataType } from "./record";
+import assert from "node:assert";
```

Comment:
Can you sort the imports?

First `node:` packages, then other packages, finally local files.

Reply:
We should figure out a way to automatically lint and fix imports :)